test/Microsoft.ML.Core.Tests/UnitTests/TestVBuffer.cs

```diff
@@ -21,5 +31,1174 @@ public void TestApplyAt()
+            VBuffer<float> a2 = new VBuffer<float>(length, a2Values);
+            var multResExpected = new float[2];
+            multResExpected[0] = aDense[3] * a2Values[3];
+            multResExpected[1] = aDense[4] * a2Values[4];
+
+            var vbufMultExpected = new VBuffer<float>(5, 2, multResExpected, new int[2] { 3, 4 });
+            var multResActual = new VBuffer<float>(length, new float[length]);
+            VectorUtils.MulElementWise(in a, ref a2);
+            VectorUtils.SparsifyNormalize(ref a2, 2, 2, normalize: false);
+            TestSame(ref vbufMultExpected, ref a2, 1e-5);
+        }
+
+        [Fact]
+        public void SparsifyNormalizeTop2()
+        {
+            var a = new VBuffer<float>(5, new float[5] { 1, 2, 3, 4, 5 });
+            var length = a.Length;
+            float[] aDense = a.DenseValues().ToArray();
+            float[] a2Values = new float[5] { -1, -2, 3, 4, 5 };
+            VBuffer<float> a2 = new VBuffer<float>(length, a2Values);
+            var multResExpected = new float[4];
+            multResExpected[0] = aDense[0] * a2Values[0];
+            multResExpected[1] = aDense[1] * a2Values[1];
+            multResExpected[2] = aDense[3] * a2Values[3];
+            multResExpected[3] = aDense[4] * a2Values[4];
+
+            var vbufMultExpected = new VBuffer<float>(5, 4, multResExpected, new int[4] { 0, 1, 3, 4 });
+            VectorUtils.MulElementWise(in a, ref a2);
+            VectorUtils.SparsifyNormalize(ref a2, 2, 2, normalize: false);
+            TestSame(ref vbufMultExpected, ref a2, 1e-5);
+        }
+
+        [Fact]
+        public void SparsifyNormalizeTopSparce()
+        {
+            for (var i = 0; i < 2; i++)
+            {
+                var norm = i != 0;
+                var a = new VBuffer<float>(7, 6, new float[6] { 10, 20, 40, 50, 60, -70 },
+                    new int[] { 0, 1, /*2 is missed*/3, 4, 5, 6 });
+                var length = a.Length;
+                float[] aDense = a.DenseValues().ToArray();
+                float[] a2Values = aDense.Select(x => x).ToArray();
+                a2Values[6] = -a2Values[6];
+                VBuffer<float> a2 = new VBuffer<float>(length, a2Values);
+                var multResExpected = new float[3];
+                multResExpected[0] = 2500;
+                multResExpected[1] = 3600;
+                multResExpected[2] = -4900;
+
+                if (norm)
+                    for (var j = 0; j < multResExpected.Length; j++)
+                        multResExpected[j] = multResExpected[j] / 4900;
+
+                var vbufMultExpected = new VBuffer<float>(7, 3, multResExpected, new int[3] { 4, 5, 6 });
+                VectorUtils.MulElementWise(in a, ref a2);
+                VectorUtils.SparsifyNormalize(ref a2, 2, 3, normalize: norm);
+                TestSame(ref vbufMultExpected, ref a2, 1e-5);
+            }
+        }
+
+        [Fact]
+        public void SparsifyNormalizeTopSparce2()
```

Comment:
I previously added some SparsifyNormalize tests (since I didn't know these tests existed).

[https://github.com/dotnet/machinelearning/blob/0fe8bafa11e0b759425100db12327569a8c444bb/test/Microsoft.ML.Core.Tests/UnitTests/TestVectorUtils.cs#L24](https://github.com/dotnet/machinelearning/blob/0fe8bafa11e0b759425100db12327569a8c444bb/test/Microsoft.ML.Core.Tests/UnitTests/TestVectorUtils.cs#L24)

Can you move those two tests here as well? So all the tests that test a specific function are in the same spot?